src/java.base/share/classes/java/lang/Class.java

```diff
@@ -642,7 +643,9 @@ private static Class<?> forName(Module module, String name, Class<?> caller) {
      * @since Valhalla
      */
     public boolean isIdentity() {
```

Comment:
The javadoc for `isIdentity` should specify if this Class is an array class, primitive type or `void`.

Reply:
The primitive classes and void also do not have specified modifier bits.
At present, they are not reported to be identity or value classes. The mainline javadoc says they are "value classes" but in reality they are identity classes. JEP 204 (future) would be the authority on that.